src/routes/+page.svelte

```diff
@@ -1,2 +1,66 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<h1>Welcome to Kendo Club!</h1>
+
+<h2>What is Kendo?</h2>
+<p>
+	Kendo is a Japanese martial art which literally translates to “The Way of the Sword”. It's an
+	athletic sport which is played by means of one-on-one striking between opponents using shinai
+	(sword made from bamboo slats) while wearing kendo-gu (protective armor).
+</p>
+
+<h2>Who are we?</h2>
+
+<p>
+	Kendo Club at the University of Michigan is led by students and instructed by highly experienced
+	coaches. Our purpose is to provide an open and inviting opportunity for University of Michigan
+	students and local community members to learn and practice Kendo. We invite everyone of any skill
+	level to join us!
+</p>
+
+<h2>How do you join?</h2>
+
+<h3>Come to practice!</h3>
+<ul>
+	<li>No prior experience needed!</li>
+	<li>
+		You may join any practice session you like (though you may find it easier to join closer to the
+		beginning of a new semester, as there are larger groups of complete beginners).
+	</li>
+	<li>Find our up-to-date practice schedule in the <a href="/calendar">Calendar</a> page.</li>
+	<li>
+		Please wear comfortable athletic clothes. Wear bottoms that cover the knee because there is some
+		kneeling involved. Kendo is a barefoot sport, so please leave your shoes outside the ballroom
+		before entering.
+	</li>
+	<li>Masking is optional. Surgical or cloth masks are the best choices for breathability.</li>
+</ul>
+
+<h3>Email us</h3>
+<ul>
+	<li>
+		You can find our email address in the <a href="/contact-us">Contact Us</a> page. We can answer additional
+		questions you have and add you to our email list (or remove you from it if you're no longer interested).
+	</li>
+</ul>
+
+<h3>Costs</h3>
+<ul>
+	<li>
+		Dues: Your first two practices are free. Afterwards, you must pay dues. These help us have space
+		to practice by contributing to practice room booking fees and organizational fees. Dues are $55
+		for one semester, and $80 for two semesters.
+	</li>
+	<li>
+		Equipment: For your first two practices, you may borrow a shinai (bamboo sword) from any club
+		member who graciously lends it to you. Afterwards, you will be offered the opportunity to
+		purchase your own equipment depending on what we have in stock. One shinai is about $30, and a
+		carrying bag is about $20.
+	</li>
+	<li>
+		Gym Membership: You are required to have a Umich rec sports membership. If you are a student
+		taking at least one class this term, it is included in your tuition. If you are a non-student,
+		consider multi-day passes or a membership. Whichever one is more economical depends on how often
+		you plan on joining practice. <a href="https://recsports.umich.edu/fitness/">
+			See the rec sports website for more details.
```

Comment:
I think only "rec sports website" needs to be in the anchor.

Reply:
Fixed in [d4b6c59](https://github.com/kendoclub-at-umich/website/pull/31/commits/d4b6c59e9299f4b6d7983b61933b97b60bb96645)